tests/third_party/cupy/linalg_tests/test_product.py

```diff
@@ -352,7 +335,6 @@ def test_transposed_tensordot_with_int_axes(self, xp, dtype):
             )
             return xp.tensordot(a, b, axes=3)
 
-    @pytest.mark.usefixtures("allow_fall_back_on_numpy")
```

Comment:
remove fixtures `@pytest.mark.usefixtures("allow_fall_back_on_numpy")` and `@testing.gpu` also for `TestProductZeroLength`